linkerd/docs/routers.md

```diff
@@ -207,3 +200,82 @@ minSize | `0` | The minimum number of connections to maintain to each host.
 maxSize | Int.MaxValue | The maximum number of connections to maintain to each host.
 idleTimeMs | forever | The amount of idle time for which a connection is cached in milliseconds.
 maxWaiters | Int.MaxValue | The maximum number of connection requests that are queued when the connection concurrency exceeds maxSize.
+
+<a name="service-configuration"></a>
+## Service Configuration
+
+This section defines the policy that linkerd will use when talking to services.
+The structure of this section depends on its `kind`.
+
+Key  | Default Value    | Description
+---- | ---------------- | --------------
+kind | `io.l5d.global` | Either [io.l5d.global](#global-service-configuration) or [io.l5d.static](#static-service-configuration)
+
+<a name="global-service-configuration"></a>
+### Global Service Config
+
+```yaml
+- protocol: http
+  service:
+    kind: io.l5d.global
+    totalTimeoutMs: 500ms
+    retries:
+      budget:
+        minRetriesPerSec: 5
+        percentCanRetry: 0.5
+        ttlSecs: 15
+      backoff:
+        kind: jittered
+        minMs: 10
+        maxMs: 10000
+```
+
+This service configuration allows you to specify [service parameters](#service-parameters)
+which will be applied to all services.
+
+<a name="static-service-configuration"></a>
+### Static Service Config
+
+```yaml
+- protocol: http
+  service:
+    kind: io.l5d.static
+    configs:
+    - prefix: /svc
+      retries:
+        budget:
+          minRetriesPerSec: 5
+          percentCanRetry: 0.5
+          ttlSecs: 15
+        backoff:
+          kind: jittered
+          minMs: 10
+          maxMs: 10000
+    - prefix: /svc/foo
+        totalTimeoutMs: 500ms
+    - prefix: /svc/bar
+        totalTimeoutMs: 200ms
+
+```
+
+This service configuration allows you to specify [service parameters](#service-parameters)
+which will be applied to all services that match a specified prefix.  The service
+configuration must contain a property called `configs` which contains a list
+of config objects.  Each config object must specify a prefix and the
+[service parameters](#service-parameters) to apply to services that match that prefix.
+If a service matches more than one config's prefix, all parameters from the
+matching configs will be applied, with parameters from later configs taking
```

Comment:
Maybe clearer to say "with parameters defined later in the configuration file taking precedence of those defined earlier" (if that's actually the case.